pkg/controller/cassandracluster/generator_test.go

```diff
@@ -130,22 +132,178 @@ func TestGenerateCassandraService(t *testing.T) {
 func TestGenerateCassandraStatefulSet(t *testing.T) {
 	assert := assert.New(t)
 
-	_, cc := helperInitCluster(t, "cassandracluster-2DC.yaml")
-	labels, nodeSelector := k8s.GetDCRackLabelsAndNodeSelectorForStatefulSet(cc, 0, 0)
-	sts := generateCassandraStatefulSet(cc, &cc.Status, "dc1", "dc1-rack1", labels, nodeSelector, nil)
-
-	assert.Equal(map[string]string{
+	expectedLabels := map[string]string{
```

Comment:
It's a matter of taste I guess. I usually do not use variables when it's used once